src/__tests__/pages/beta/[viewType]/curriculum/index.test.tsx

```diff
@@ -35,13 +53,10 @@ describe("pages/beta/curriculum/index", () => {
     );
   });
 
-  describe("getStaticProps", () => {
-    it("Should fetch the correct data", async () => {
-      const testRes = (await getStaticProps({})) as {
-        props: CurriculumHomePageProps;
-      };
-      expect(testRes.props.subjectPhaseOptions).toEqual(subjectPhaseOptions);
-    });
+  it("Renders curriculum blogs", () => {
+    render(<CurriculumHomePage {...props} />);
+    const blogList = screen.getByTestId("blog-list");
+    expect(blogList).toBeInTheDocument();
```

Comment:
Looks good, though it might be good to add a test for each blog item and an expected count, eg:
```
const blogPosts = screen.getAllByTestId("blog-item");
expect(blogPosts).toHaveLength(3);
```

Reply:
test added